docs/CMakeLists.txt

```diff
@@ -80,9 +80,9 @@ set(doxygen_dependencies
     "${hpx_SOURCE_DIR}/hpx/parallel/algorithms/uninitialized_copy.hpp"
     "${hpx_SOURCE_DIR}/hpx/parallel/algorithms/uninitialized_fill.hpp"
     "${hpx_SOURCE_DIR}/hpx/runtime/get_ptr.hpp"
-    "${hpx_SOURCE_DIR}/hpx/runtime/actions/action_support.hpp"
+    "${hpx_SOURCE_DIR}/hpx/runtime/actions/basic_action.hpp"
```

Comment:
The bits from `action_support.hpp` that contribute to the documentation have been moved to `basic_action.hpp` about two commits ago.